cypress/integration/AppAnalytics/DiseaseOnly/DiseaseAnalytics.feature

```diff
@@ -22,3 +22,18 @@ Feature: Disease listing page analytics
 			| page.additionalDetails.diseaseName          | breast cancer                                             |
 			| page.additionalDetails.trialListingPageType | disease                                                   |
 			| page.additionalDetails.numberResults        | (int)938                                                  |
+
```

Comment:
As discussed add the line when rebasing.

Reply:
done

Reply:
Actually, this line needs to be added to the below test, with the click event - it's already been added to the first test with the previous commit to develop.